phys/module_radiation_driver.F

```diff
@@ -2020,6 +2054,7 @@ SUBROUTINE radiation_driver (                                  &
            tlwup(i,j) = erbe_out(i,j,2)    ! TOA LW upwelling flux [W/m2]
            slwdn(i,j) = erbe_out(i,j,3)    ! surface LW downwelling flux [W/m2]
            slwup(i,j) = erbe_out(i,j,4)    ! surface LW upwelling flux [W/m2]
+           olr(i,j)   = -erbe_out(i,j,2)
```

Comment:
is erbe_out(i,j,2) negative? This negative sign is only needed if it is.

Reply:
@dudhia Yes, it is negative. I plotted the original first, but the values are all negative.